backend/src/main/java/eliasyazdani/capstone/cupwithme/backend/player/PlayerService.java

```diff
@@ -0,0 +1,30 @@
+package eliasyazdani.capstone.cupwithme.backend.player;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+
+@Service
+public class PlayerService {
+    private final PlayerRepository playerRepository;
+
+    public PlayerService(PlayerRepository playerRepository) {
+        this.playerRepository = playerRepository;
+    }
+
+    public List<PlayerWithOrdinalNumber> getAllPlayers() {
+        List<Player> playerFromDB = playerRepository.findAll();
+        List<PlayerWithOrdinalNumber> playerWithOrdinalNumbers = new ArrayList<>();
+        for (int i = 0; i < playerFromDB.size(); i++) {
+            Player player = playerFromDB.get(i);
+            playerWithOrdinalNumbers.add(new PlayerWithOrdinalNumber(
+                    i + 1, player.firstName(), player.lastName(), player.age()));
+        }
```

Comment:
You can consider using the following syntax, it's more clean:
```suggestion
     int ordinalNumber = 1;
     for (Player player : playerFromDB) {
    playerWithOrdinalNumbers.add(new PlayerWithOrdinalNumber(
        ordinalNumber, player.firstName(), player.lastName(), player.age()));
    ordinalNumber++;
}
```